system/Validation/Validation.php

```diff
@@ -506,6 +500,8 @@ public function setRuleGroup(string $group)
 
     /**
      * Returns the rendered HTML of the errors as defined in $template.
+     *
+     * You can also use validation_list_errors() in Form helper.
```

Comment:
It is certainly possible to deprecate this method since it duplicates functionality. 
If so, what would we do with the error view files and templates setting?
See https://codeigniter4.github.io/CodeIgniter4/libraries/validation.html#configuration

Reply:
It looks to me like your functions still use those components. What else would you do with them?

Reply:
My question is: If we decouple View from the Validation class, where do we put the validation error templates and its configuration?

Reply:
Counterquestion. Why are error patterns needed?

Reply:
What's error pattern?